tests/all_mode/run.sh

```diff
@@ -32,14 +34,29 @@ function run() {
     run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $cur/conf/dm-worker1.toml
     run_dm_worker $WORK_DIR/worker2 $WORKER2_PORT $cur/conf/dm-worker2.toml
 
+    # kill tidb
+    pkill -hup tidb-server 2>/dev/null || true
+    wait_process_exit tidb-server
+
     run_sql_file $cur/data/db1.increment.sql $MYSQL_HOST1 $MYSQL_PORT1
     run_sql_file $cur/data/db2.increment.sql $MYSQL_HOST2 $MYSQL_PORT2
 
+    sleep 2
+    # dm-worker execute sql failed, and will try auto resume task
+    check_log_contains $WORK_DIR/worker1/log/dm-worker.log "dispatch auto resume task"
+    check_log_contains $WORK_DIR/worker2/log/dm-worker.log "dispatch auto resume task"
+
+    # restart tidb, and task will recover success
+    run_tidb_server 4000
+    sleep 1
```

Comment:
Is it possible more time needed to wait?

Reply:
check_sync_diff will retry several times, so it doesn't matter. and I increase to 2